addons/udes_get_info/models/models.py

```diff
@@ -10,13 +10,16 @@
 
 
 @add_if_not_exists(models.BaseModel)
-def _get_info(self, level=0, max_level=None, info_fields=None, extra_fields=None):
-
+def _get_info(self, level=1, max_level=None, info_fields=None, extra_fields=None):
+    """Returns the information of the current object in the case when level equals to max_level
+    it will return only BASIC_GET_INFO_VALUES of any relational field of current object.
+    In the case level is less than max_level it will return the specified fields of any
+    relational fields of the current object."""
```

Comment:
Perhaps: "Returns the information of the current object. If level equals max_level
     it will return only BASIC_GET_INFO_VALUES of any relational field of the current object.
     If level is less than max_level it will return the specified fields of any
     relational fields of the current object."